.github/release.yml

```diff
@@ -0,0 +1,17 @@
+# .github/release.yml
+changelog:
+  # exclude:
+  #   labels:
+  #     - ignore-for-release
+  #   authors:
+  #     - 
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - BREAKING CHANGE
+    - title: Exciting New Features 🎉
+      labels:
+        - feat
+    - title: Other Changes
+      labels:
+        - "*"
```

Comment:
Are these categories tied to the versioning system (with major, minor and patches)? 

As @AngryMaciek points to a page [in this comment](https://github.com/zavolanlab/zarp/issues/5#issuecomment-931569324), I assume that it can be independent. 
A combination of commits in (possibly) independent PRs constitute a new version, based on [this site](https://www.conventionalcommits.org/en/v1.0.0/).

So I guess we could expand "Other Changes" a bit, at least to include `fix`, `chore`, `docs`, `perf` (I assumed performance), and `test`. Would that be feasible?

Reply:
Thanks for the comments @dominikburri! Essentially, using the fix, chore, docs, perf are all by default under the "other changes" since I am using "*".  But this also means that non standard options can be used. This is why in the PR I added guidelines on how to write the PR.